src/Utilities/Utilities.jl

```diff
@@ -501,11 +501,25 @@ function linerange(text, from)
     return lines > 0 ? (from:(from + lines + 1)) : (from:from)
 end
 
-function format_line(range::AbstractRange)
-    top = format_line(first(range))
-    return length(range) <= 1 ? top : string(top, '-', format_line(last(range)))
+struct LineRangeFormatting
+    prefix::String
+    separator::String
+
+    function LineRangeFormatting(repoURL::String)
+        if (startswith(repoURL, "https://bitbucket.org") || startswith(repoURL, "http://bitbucket.org"))
```

Comment:
I am wondering whether there is a cleaner way to do this. Does anyone know of whether we can easily parse a URL/URI with julia's base packages?

Reply:
I just found this: https://github.com/JuliaDocs/Documenter.jl/blob/181a1b6bbded1904ef69767d0312ed7619bd448a/src/Writers/HTMLWriter.jl#L411 . so we could use `contains`, although it may match latter parts of the URI and lead to an unwanted behavior

Reply:
I reckon it is probably good enough, so it would be fine if we stick to that. But I think it would be good if this check would be factored out into a separate function though (maybe returning `:github`, `:bitbucket`, `:gitlab`?), so that we wouldn't duplicate that logic.

Reply:
@mortenpi makes sense, I'll make the changes and ping you when done. Would you place that functionality in `Utilities.jl` and use it from other modules like `HTMLWriter.jl`? To know where to place them already.

Reply:
> Would you place that functionality in `Utilities.jl` and use it from other modules like `HTMLWriter.jl`?

Yup